pkg/operation/botanist/component/fluentoperator/crds/assets/crd-fluentbit.fluent.io_clusterfluentbitconfigs.yaml

```diff
@@ -0,0 +1,275 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    gardener.cloud/deletion-protected: "true"
```

Comment:
Does this deletion protection work out of the box? Generally, I don't think we need this for the fluent-bit CRDs. We only use this for critical resources like ETCD or extensions.